### ISSUE TITLE
Underlord does not automatically channel teleport upon casting Fiend's Gate

### ISSUE THREAD

**Issue by gRyanL** [ValveSoftware/Dota2-Gameplay#8072]
### Ability name

Fiend's Gate

### Description

Since the patch, the Underlord does not automatically use the gate upon casting Fiend's Gate, meaning that I have to navigate back to my hero and manually move up to the gate to actually start channeling.

[Reddit Post](https://www.reddit.com/r/DotA2/comments/12tr6tx/underlord_does_not_automatically_channel_teleport/)

### Example Match ID (and possibly Timestamp)

7116674990 - 04:30

**Comment by commentator24** [comment]
yeah kinda annoying to ult somewhere then have to click the portal

**Comment by guianselmi** [comment]
Yeah, it was not documented as change, and it causes bad experiences in the gameplay.

**Comment by guianselmi** [comment]
It looks fixed now.
